### PR TITLE
Add alpha and beta references to README.rst.

### PR DESCRIPTION
**Pull request by daspecster**
Towards #2820.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -15,23 +15,28 @@ Google Cloud Python Client
...
--  `Google Cloud Bigtable`_ (`Bigtable README`_)
--  `Google Cloud DNS`_ (`DNS README`_)
--  `Stackdriver Error Reporting`_ (`Error Reporting README`_)
--  `Google Cloud Natural Language`_ (`Natural Language README`_)
--  `Google Translate`_ (`Translate README`_)
--  `Google Cloud Vision`_ (`Vision README`_)
--  `Google Cloud Bigtable - HappyBase`_ (`HappyBase README`_)
--  `Google Cloud Runtime Configuration`_ (`Runtime Config README`_)
+
+This client supports the following Google Cloud Platform services at a Beta_ quality level:
```
Comment (dhermes) [inline README.rst]: Beta here is not a valid RST link?

File: README.rst [hunk 2]
```diff
@@ -15,23 +15,28 @@ Google Cloud Python Client
...
+
+-  `Google BigQuery`_ (`BigQuery README`_) (Beta)
+-  `Google Cloud Datastore`_ (`Datastore README`_) (Beta)
+-  `Stackdriver Logging`_ (`Logging README`_) (Beta)
+-  `Google Cloud Storage`_ (`Storage README`_) (Beta)
```
Comment (dhermes) [inline README.rst]: 1. The extra "(Beta)" is redundant
1. This list is missing `Pub / Sub`

File: README.rst [hunk 3]
```diff
@@ -15,23 +15,28 @@ Google Cloud Python Client
...
+-  `Google Cloud Resource Manager`_ (`Resource Manager README`_) (Alpha)
+-  `Stackdriver Monitoring`_ (`Monitoring README`_) (Alpha)
+-  `Google Cloud Bigtable`_ (`Bigtable README`_) (Alpha)
+-  `Google Cloud DNS`_ (`DNS README`_) (Alpha)
+-  `Stackdriver Error Reporting`_ (`Error Reporting README`_) (Alpha)
+-  `Google Cloud Natural Language`_ (`Natural Language README`_) (Alpha)
+-  `Google Translate`_ (`Translate README`_) (Alpha)
+-  `Google Cloud Vision`_ (`Vision README`_) (Alpha)
+-  `Google Cloud Bigtable - HappyBase`_ (`HappyBase README`_) (Alpha)
+-  `Google Cloud Runtime Configuration`_ (`Runtime Config README`_) (Alpha)
```
Comment (dhermes) [inline README.rst]: Ditto on "(Alpha)" being redundant

File: README.rst [hunk 4]
```diff
@@ -97,6 +102,22 @@ You may also find the `authentication document`_ shared by all the
 .. _Authentication section: https://google-cloud-python.readthedocs.io/en/latest/google-cloud-auth.html
 .. _authentication document: https://github.com/GoogleCloudPlatform/gcloud-common/tree/master/authentication
 
+.. _Alpha: #versioning
+.. _Beta: #versioning
+
+Versioning
```
Comment (dhermes) [inline README.rst]: This section doesn't belong here. We should just link to the GCP Alpha/Beta distinction (if it exists). @jonparrott?

File: README.rst [hunk 5]
```diff
@@ -15,23 +15,28 @@ Google Cloud Python Client
...
--  `Google Cloud Runtime Configuration`_ (`Runtime Config README`_)
+
+This client supports the following Google Cloud Platform services at a Beta_ quality level:
+
+-  `Google BigQuery`_ (`BigQuery README`_) (Beta)
+-  `Google Cloud Datastore`_ (`Datastore README`_) (Beta)
+-  `Stackdriver Logging`_ (`Logging README`_) (Beta)
+-  `Google Cloud Storage`_ (`Storage README`_) (Beta)
+
+
```
Comment (dhermes) [inline README.rst]: Again, extra newline for fun?

File: README.rst (hunk 2 above)
Comment (tseaver) [inline README.rst]: Did I miss a discussion which decided that pubsub was not one of the "golden set" going beta?

Comment (daspecster) [inline README.rst]: I was going off the screenshots for #2820. I thought that was strange as well.

@dhermes I'm not sure if they want these to match the other repos or not but maybe we could get a little more clarification from @omaray?

File: README.rst (hunk 1 above)
Comment (daspecster) [inline README.rst]: http://docutils.sourceforge.net/docs/user/rst/quickref.html#internal-hyperlink-targets

File: README.rst [hunk 6]
```diff
@@ -15,23 +15,28 @@ Google Cloud Python Client
...
--  `Stackdriver Monitoring`_ (`Monitoring README`_)
--  `Google Cloud Bigtable`_ (`Bigtable README`_)
--  `Google Cloud DNS`_ (`DNS README`_)
--  `Stackdriver Error Reporting`_ (`Error Reporting README`_)
--  `Google Cloud Natural Language`_ (`Natural Language README`_)
--  `Google Translate`_ (`Translate README`_)
--  `Google Cloud Vision`_ (`Vision README`_)
--  `Google Cloud Bigtable - HappyBase`_ (`HappyBase README`_)
--  `Google Cloud Runtime Configuration`_ (`Runtime Config README`_)
+
```
Comment (daspecster) [inline README.rst]: Open floor plan?

File: README.rst (hunk 2 above)
Comment (daspecster) [inline README.rst]: https://github.com/GoogleCloudPlatform/google-cloud-python/pull/2821/commits/76d5efbd57e8cded5600caa4babe9c6418d603fb

File: README.rst (hunk 4 above)
Comment (daspecster) [inline README.rst]: There's no mention of semver in that link though.

Comment (daspecster) [inline README.rst]: https://github.com/GoogleCloudPlatform/google-cloud-python/pull/2821/commits/bc1ae8df3ba3afed256baee0dc6fa38d4ebb0b3e

File: README.rst [hunk 7]
```diff
@@ -70,6 +70,8 @@ This client supports the following Google Cloud Platform services at an Alpha_ q
 If you need support for other Google APIs, check out the
 `Google APIs Python Client library`_.
 
+.. _Alpha: https://cloud.google.com/terms/launch-stages
+.. _Beta: https://cloud.google.com/terms/launch-stages
```
Comment (dhermes) [inline README.rst]: What's the point of having two things point to the same link? (Honest question, feels like it's not worthwhile, but maybe I'm wrong.)

Comment (daspecster) [inline README.rst]: I could remove the reference from

>This client supports the following Google Cloud Platform services at a Beta_ quality level:

then add something like this at the bottom of the main section.

>If you would like to learn more about the launch stages for the
Google Cloud Platform please visit https://cloud.google.com/terms/launch-stages.

**Comment by omaray** [comment]
Hey guys. So the Beta/Alpha links you gave was a good attempt! But these apply to the Services in Google Cloud (and doesn't apply to client libraries really). So for now the idea is to just add what "Beta" and "Alpha" means in the Versioning section. We can always change later or have a centralized location in gcloud-common or something. But at least it gives a succinct explanation. You can see the PR for PHP as a reference: https://github.com/GoogleCloudPlatform/google-cloud-php/pull/274 (which you probably already have).


**High Priority**: Pub/Sub is NOT Beta. Please we need to put in the Alpha section!

**Comment by daspecster** [comment]
@omaray I moved Pub/Sub to the Alpha section and put the Version section back.

Let me know if this looks good to you?

**Comment by tseaver** [comment]
@omaray Out of curiousity, what is the rationale for holding Pub/Sub as "alpha"?  I don't recall any discussion of it, and we've been talking about the "fab five" for months.

**Comment by dhermes** [comment]
@daspecster @omaray FWIW, whether Pub/Sub is part of some big picture or not, we are treating it (local to Python) like it is "beta" quality.

**Comment by daspecster** [comment]
@dhermes @omaray what if we changed the verbage to something like...
```diff
+ This client supports the following Google Cloud Platform services which are at a Beta_ quality level
- This client supports the following Google Cloud Platform services at a Beta_ quality level
```

File: README.rst [hunk 8]
```diff
@@ -15,14 +15,21 @@ Google Cloud Python Client
 .. _API Documentation: https://googlecloudplatform.github.io/google-cloud-python/stable/
 .. _Read The Docs Documentation: https://google-cloud-python.readthedocs.io/en/latest/
 
-This client supports the following Google Cloud Platform services:
+This client supports the following Google Cloud Platform services at **Beta**:
```
Comment (jonparrott) [inline README.rst]: `at Beta` seems like weird phrasing.

Perhaps:

```
Client library support for the following Google Cloud Platform services is in **beta**:
```

or

```
This client library has **beta** support for the following Google Cloud Platform services:
```

File: README.rst [hunk 9]
```diff
@@ -15,14 +15,21 @@ Google Cloud Python Client
...
 
+-  `Google BigQuery`_ (`BigQuery README`_)
 -  `Google Cloud Datastore`_ (`Datastore README`_)
+-  `Stackdriver Logging`_ (`Logging README`_)
 -  `Google Cloud Storage`_ (`Storage README`_)
+
+Libraries defined as **Beta** quality are expected to be mostly stable and
```
Comment (jonparrott) [inline README.rst]: Try not to use we.

Perhaps:

```
**Beta** indicates that the client library for a particular service is mostly stable and is being prepared for release. Issues and requests against beta libraries are addressed with a higher priority.
```

File: README.rst [hunk 10]
```diff
@@ -15,14 +15,21 @@ Google Cloud Python Client
...
+we're working towards their release candidate. We will address issues and
+requests with a higher priority.
+
+This client supports the following Google Cloud Platform services at **Alpha**:
```
Comment (jonparrott) [inline README.rst]: Same comments here re: the beta sentence.

**Pull request merged**